xla/backends/gpu/codegen/custom.cc

```diff
@@ -266,26 +257,18 @@ std::unique_ptr<HloModule> ExtractOffsetModule(
 // these conditions are ensured before fusing in
 // `dynamic_slice_fusion_rewriter.cc:IsValueFunctionOfLoopInductionVariable`.
 std::unique_ptr<HloModule> ExtractWhileUpdateModule(
-    const HloInstruction* while_op) {
```

Comment:
Could you please update the docstring to explain what indvar_idx corresponds to?